x-pack/plugins/maps/public/plugin.ts

```diff
@@ -97,6 +96,8 @@ export class MapsPlugin
   }
 
   public setup(core: CoreSetup, plugins: MapsPluginSetupDependencies) {
+    registerLicensedFeatures(plugins.licensing);
```

Comment:
Does this call need `if plugins.licensing` check like the one in `start`?

Reply:
I'll go ahead and remove the if-check . This is an x-pack plugin, where licensing should also be present. Looking at other plugins, they do not check if any of the start-services exist either. Not sure why the if-check was there in the  first place.